lib/websocket.js

```diff
@@ -1,22 +1,69 @@
 'use strict';
 
 var levels = require('./levels');
-
+var times = require('./times');
+var calcData = require('./data/calcdelta');
+var ObjectID = require('mongodb').ObjectID;
+var profilefunctions = require('./profilefunctions')();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["profilefunctions" is defined but never used](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2651207102/issues/source?bid=2452374&fileBranchId=3324571#l7)